client/src/components/Home.jsx

```diff
@@ -107,11 +109,20 @@ const Home = ({ setIsLoading, isMobile, isLoggedIn, isLoading, setIsLoggedIn, sh
   }
 
   // POST '/todoList/:userEmail' -> Adding or Upserting a "todoList item"
+  //modified to use actual user email
   const addTodo = (todo) => {
     console.log('Add todo: ', todo);
-    axios.post('http://localhost:3000/category/todoList', { params: { userEmail: userEmail }, data: todo })
+    const incomingEmail = info.user_email;
+    axios.post('http://localhost:3000/todoList/item', { params: { userEmail: incomingEmail }, data: todo })
       .then((result) => {
         console.log(result);
+        let catId = result.data.id;
+        console.log('all todos before: ', myEvents);
+        let newTask = {item_id: catId, title: todo.title, description: todo.description, duration: todo.duration, start: todo.start, end_time: todo.end_date, in_calendar: todo.in_calendar};
```

Comment:
maybe write out this line into separate lines with key-value pairs?

Reply:
For readability, as it is a long object.

Reply:
Absolutely. Easy refactor.